corporate/urls.py

```diff
@@ -4,12 +4,19 @@
 from django.urls import path
```

Comment:
Should all the URLs here have an ending slash?

Reply:
Probably a good topic to discuss in Zulip; I'm not sure what is best.